modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -125,10 +126,21 @@ public function build() {
     });
 
     $count = 0;
-    $cart_views = [];
+    $content = [];
     if (!empty($carts)) {
-      $cart_views = $this->getCartViews($carts);
+      $order_type_storage = $this->entityTypeManager->getStorage('commerce_order_type');
       foreach ($carts as $cart_id => $cart) {
+        if ($this->configuration['dropdown']) {
+          $order_type = $cart->bundle();
+          $order_type_entity = $order_type_storage->load($order_type);
```

Comment:
We should follow the `$order_types = $order_type_storage->loadMultiple(array_unique($order_type_ids));` pattern. `loadMultiple` is really that much more performant.